db.php

```diff
@@ -0,0 +1,10 @@
+<?php
+  $servername = 'localhost';
+  $username = 'root';
+  $password = '';
+  $dbname = "my_db";
+  $conn = mysqli_connect($servername, $username, $password, "$dbname");
+    if(!$conn){
+      die('Could not connect MySql Server: ' .mysql_error());
```

Comment:
```suggestion
      die('Could not connect MySql Server: ' . mysqli_error());
```
- Space after `.` operator
- https://www.php.net/mysql_error/ is deprecated, use https://www.php.net/manual/en/mysqli.error.php